guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -85,23 +85,36 @@ ifdef::katello[]
 ----
 # foreman-rake katello:upgrade_check
 ----
-. Update operating system packages
+. Update operating system packages:
 +
 [options="nowrap" subs="attributes"]
 ----
 # yum update -y
 ----
 . Update repositories
 +
-.For Centos 7 or {RHEL} Users:
+.For Centos 7 or {RHEL} 7 Users:
 [options="nowrap" subs="attributes"]
 ----
-# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
-# yum update -y https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm \
+                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
 # yum install -y centos-release-scl-rh
 ----
-. Update packages
-Clean the yum cache and update the required packages:
++
+.For Enterprise Linux 8 or {RHEL} 8 Users:
+[options="nowrap" subs="attributes"]
+----
+# dnf update -y https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
+                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+----
+// This step is only needed for upgrades to Katello 4.5 and later
+. Make sure the Module Streams are enabled for Enterprise Linux 8 and {RHEL} 8:
```

Comment:
```suggestion
. Ensure the Module Streams are enabled for Enterprise Linux 8 and {RHEL} 8:
```

I don't know for sure about capitalization of "module streams".

Reply:
I always wondered what the correct form to write the streams is, thank you for the correction!